### ISSUE TITLE
Running polaris in CI pipeline with flag set-exit-code-below-score 90 always fails

### ISSUE THREAD

**Issue by geetuBalwani** [FairwindsOps/polaris#558]
### Installation Process
Using polaris 4.0.2 inside Jenkins pipeline creating a docker container with debian base image.

### Expected Behavior
Running
```
polaris audit --audit-path /tmp/workspace/lint-check --format=pretty --color=false --set-exit-code-below-score 90 --set-exit-code-on-danger --only-show-failed-tests true
```
should have resulted in jenkins build failure if the score was less than 90.

### Actual Behavior
Irrespective of the score build fails with an info message in logs "Audit score of 0 is less than the provided minimum of 90"
whereas while auditing score is printed and in the given case this is 94
Polaris audited Path /tmp/workspace/lint-check at 2021-06-03T09:55:28Z
    Nodes: 0 | Namespaces: 0 | Controllers: 4
    Final score: 94

### Steps to Reproduce
polaris audit --audit-path /tmp/workspace/lint-check --format=pretty --color=false --set-exit-code-below-score 90 --set-exit-code-on-danger --only-show-failed-tests true

**Comment by rbren** [comment]
This is a strange one! Thanks for reporting

Here is a command to reproduce using the polaris repo alone:

```
go run main.go audit --audit-path ./test/webhook_cases/ --format=pretty --color=false --set-exit-code-below-score 70 --only-show-failed-tests true
```

We'll take a look shortly

**Comment by rbren** [comment]
This is fixed in 4.0.3

Turns out it was an interaction between `--set-exit-code-below-score` and `--only-show-failed-tests`
